app/services/data_request.rb

```diff
@@ -0,0 +1,73 @@
+require 'net/http'
+
+class DataRequest
+  attr_reader :data
+
+  # Set arguments here
+  def initialize(data)
+    @data = data
+  end
+
+  # * means capture all arguments and keep them in an array called "args"
+  def self.call(*args)
+    # This calls initialize, converting the args array back into arguments
+    self.new(*args).call
+  end
+
+  # Run logic here
+  def call
+    # parsed_json["results"].each do |event_hash|

```

Comment:
Can we clean out some of this commented code?

Reply:
Yes we can.